src/context/index.tsx

```diff
@@ -17,7 +17,7 @@ const AppWrapperContext: FC<Props> = ({ children }) => {
     const { edit: editQuery } = query;
     const isEditMode = editQuery === 'true';
     const onEditRoute = () => {
-        router.replace('?edit=true'); //TODO handle query better https://github.com/Real-Dev-Squad/website-status/issues/299 
+        router.push({ query: "edit=true" }, undefined, { shallow: true }); //TODO handle query better https://github.com/Real-Dev-Squad/website-status/issues/299 
```

Comment:
the code is not linted properly, run the `lint-fix` script and then push it

Reply:
ok will do that.

Reply:
Hi @ivinayakg I have fixed the changes can u check it once.

Reply:
have you ran the `lint-fix` script?

Reply:
yes